website/docs/r/managed_policies.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Policies"
+layout: "lacework"
+page_title: "Lacework: lacework_managed_policies"
+description: |-
+  Manage Lacework Defined Policies
+---
+
+# lacework\_managed\_policies
+
+Use this resource to update the `state` (enabled/disabled) and the `severity` properties for Lacework-defined policies.
+
+## Example Usage
+
+The following example shows how to manage three Lacework-defined policies.
+
+```hcl
+resource "lacework_managed_policies" "example" {
+  policy {
+    id       = "lacework-global-1"
+    enabled  = true
+    severity = "High"
+  }
+  policy {
+    id       = "lacework-global-2"
+    enabled  = false
+    severity = "Critical"
+  }
+  policy {
+    id       = "lacework-global-10"
+    severity  = "Low"
```

Comment:
enabled is required, this example is wrong
```suggestion
    severity  = "Low"
    enabled = true
```

add enabled and please format (align) this example code